packages/saber-plugin-image/lib/saber-browser.js

```diff
@@ -13,16 +13,19 @@ export default ({ Vue }) => {
 
       if ($el.dataset.src || $el.dataset.srcset) {
         lozad($el, {
-          loaded(el) {
-            el.addEventListener(
-              'load',
-              () => el.setAttribute('data-lazy-loaded', ''),
```

Comment:
/cc @kidonng, I'm not sure why you were using this instead of the `data-loaded` attribute which is added by lozad itself.

Reply:
https://github.com/h404bi/www.h404bi.com/commit/afb8fb85172f7194b78449e3529651ca85fec2fb#r34409511

Reply:
Ah you're right.